README.md

```diff
@@ -144,12 +144,14 @@ In the above examples we are making use of Chordly's `options` object which may
   bind new sequence(s)
   args[0] array of sequence/lookup/matched objects to add to the sequenceMap
 
+```js
           $(expression).chordly('bind', {
               sequence: $.chordly.literalStringToSequence('mouse'),
               matched: function () {
                   console.log('squeak!')
               }
           });
```

Comment:
Fix the indentation of this too please. It should not be prefixed with so much space. Something like

```markdown
```js
$(expression).chordly('bind', {
   sequence: $.chordly.literalStringToSequence('mouse'),
   matched: function () {
      console.log('squeak!')
   }
});
```
```

(hmmm, I can't see to properly put a markdown code example within markdown)

Reply:
I wasn't sure about the indentation because it looked intentional for keeping it inline with the examples above and below. But it does look out of place with the other code blocks.